src/utils/operations.ts

```diff
@@ -27,19 +24,32 @@ export const getDataFromEvent = <T>(event: AnyEvent, field: string, possibleInde
   }
 
   if (index === undefined || index === -1) return;
-  
-  return event.data[index] as T;  
-}
+
+  return event.data[index] as T;
+};
+
+const MULTISIG_EVENTS = ["NewMultisig", "MultisigApproval", "MultisigExecuted", "MultisigCancelled"];
+export const getCallHashFromMultisigEvents = (events: AnyEvent[]) => {
+  const multisigEvent = events.find(e => MULTISIG_EVENTS.includes(e.method));
```

Comment:
visited call may contains a few the same events it is not garantee so the first one found may not be the one you need.

Reply:
This method seems unused